interface/forms/newpatient/common.php

```diff
@@ -491,6 +491,12 @@ function cancelClickedOld() {
                                 ?>
                             </select>
                         </div>
+                        <div class="col-sm-2">
+                            <label for='class' class="oe-text-to-right"><?php echo xlt('Class'); ?>:</label>
+                        </div>
+                        <div class="col-sm">
+                            <?php echo generate_select_list('class_code', '_ActEncounterCode', $viewmode ? $result['class_code'] : 'AMB', ''); ?>
```

Comment:
hi @yashrajbothra , Would do this instead:
```
<?php echo generate_select_list('class_code', '_ActEncounterCode', $viewmode ? $result['class_code'] : '', '', ''); ?>
```
(in this case the list itself will enforce the default and that empty 5th parameter will remove the "empty" selection)